### DISCUSSION TITLE
Développement application mobile

### DISCUSSION THREAD

**Discussion by VahanLab** [hostolab/covidliste#491] [Ideas]
Bonjour à toute l'équipe,

Je me présente Karl Gochgarian, je suis développeur spécialisé dans les application mobiles natives (iOS & Android).
Je trouve votre outil génial pour la lutte contre le covid-19. Bravo pour votre initiative et la mise en oeuvre 👏.

J'aimerais beaucoup pouvoir contribuer à l'effort de guerre. Votre plateforme web permet déjà de répondre au besoin, mais si vous voyez un intérêt à mettre également à disposition une application mobile, je suis chaud à 100% pour endosser son développement.

 Qu'en dites-vous ?

**Comment by FelixMalfait** [comment]
Salut @VahanLab ! Merci pour la proposition c'est super sympa.
Je ne pense pas qu'il y ait de besoin iOS ou Android à ce stade. Si jamais tu veux faire du frontend ou du Ruby en revanche, on ne manque pas de problème à résoudre ;).

**Reply by VahanLab** [reply]
Hello ! 
Ok ça marche, n'hésitez pas si vous changez d'avis 🙂 
Ca aurait été avec plaisir, mais malheureusement je ne maitrise pas les technos de votre stack donc je ne serai pas d'une grande aide !

**Comment by psa-jforestier** [comment]
Une solution alternative au développement d'une app (et à la publication de l'app sur les stores), est d'utiliser les fonctions du navigateur pour ajouter le site web à la page d’accueil. Cela peut aider ceux qui ne maitrisent pas l'option similaire présente dans le navigateur .
https://github.com/cubiq/add-to-homescreen
https://medium.com/@sam20gh/how-to-add-add-to-home-screen-to-your-website-4b07aee02676
